packages/interoperator/pre_packaging

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
By any chance, is it possible to avoid using pre_packaging ?
According to [this](https://bosh.io/docs/create-release/#pkg-skeletons)  

`Use of the pre_packaging file is not recommended, and is not discussed in this tutorial.`